packages/ilios-common/addon/models/user.js

```diff
@@ -300,6 +300,22 @@ export default class User extends Model {
     return false;
   }
 
+  /**
+   * Checks if this user is instructing any offerings or ILMs.
+   */
+  get isInstructor() {
```

Comment:
This doesn't work if someone is in an instructor group attached to one of these things. I think the existing `allInstructedCourses` property may have what you want already.

Reply:
i think i gave up on this idea in April, then promptly forgot about it after vacation. horray for code reviews! 

removed again.  :heavy_check_mark: